tests/duplicate-instance.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from "./fixtures/lxd-test";
+import { deleteAllImages } from "./helpers/images";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitInstance,
+} from "./helpers/instances";
+
+test("Duplicate an instance", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+
+  await page.goto("/ui/");
+  await visitInstance(page, instance);
+  await page.getByRole("button", { name: "Duplicate" }).click();
+  await page
+    .getByRole("dialog", { name: "Duplicate Instance" })
+    .getByRole("button", { name: "Duplicate" })
+    .click();
+
+  await page.waitForSelector(`text=Created instance ${instance}-duplicate.`);
+
+  await deleteAllImages(page);
+  await deleteInstance(page, instance);
```

Comment:
```suggestion
```
When using the shared instance, we don't need to clean up.
But we do need to remove the duplicate instance that gets created above.